app/views/results.nunjucks

```diff
@@ -23,16 +23,25 @@
     {% include "includes/search-form.nunjucks" %}
 
   {% else %}
-
-    <a href="{{ SITE_ROOT }}" class="link-back">Back</a>
-    <h1 class="sr-only" data-results="{{gps.length}}">Search results</h1>
-    <h2 class="results__header">GP {% if gps.length === 1 %}surgery{% else %}surgeries{%endif %} matching '{{ search }}' </h2>
-    <p>If you can't see your surgery, check that you're using the right name and try searching again.</p>
+    <h1 class="sr-only" data-results="{{gps.length}}" data-term="{{ search }}">Search results</h1>
+    {% if gps.length === 1 %}
+      <h2 class="results__header">Is this your surgery?</h2>
+    {% else %}
+      <h2 class="results__header">Which is your surgery?</h2>
+    {% endif %}
+    <p>We found
```

Comment:
Can this logic be done outside of the view so there is just a single property to render, like the code above could be turned into?